DietPi-Software | myMPD

Joulinar opened the pull request:
**Status**: Ready
- [x] adjust `dietpi-software`

**Reference**: https://github.com/MichaIng/DietPi/issues/4256

**Commit list/description**:
- DietPi-Software | myMPD - adjust web server configuration to reflect new http port variable

Joulinar committed:
v7.1

DietPi-Software | myMPD - adjust web server configuration to reflect new http port variable

MichaIng committed:
v7.1

+ DietPi-Software | myMPD: Relax HTTP port config injection to what it was before and assure that it is added to the [webserver] section

MichaIng commented:
Thanks, good find. I re-added the allowed missing or multiple spaces after the config name and that it is assured to be added to the still correct `[webserver]` section.

MichaIng committed:
v7.1

+ CHANGELOG | myMPD: Resolved an issue where the service fails to start because of a renamed setting
